### DISCUSSION TITLE
How to fix ENOTEMPTY Error when starting my development server

### DISCUSSION THREAD

**Discussion by Segun98** [vercel/next.js#20259] [Help]
I recently started getting an ENOTEMPTY when I start my dev server (npm run dev). How to fix please?

## Screenshots

![Screenshot (34)](https://user-images.githubusercontent.com/53624548/102484274-4eee1600-4066-11eb-9d30-792640f22e23.png)

I *think* it started after I installed a new React app in an another folder entirely different from where this Nextjs project is located

- OS:  Windows
- Version of Next.js: 10.0.1
- Version of Node.js: 12.0.0
- Deployment: Vercel

**Comment by Segun98** [comment]
Somebody help :(

**Comment by repercussive** [comment]
if anyone has this problem in the future, it can be fixed by deleting the .next folder and allowing Next.js to regenerate it.
